Project.toml

```diff
@@ -13,6 +14,7 @@ ITensorMPS = "0.1, 0.2"
 ITensors = "0.3.18, 0.4, 0.5, 0.6"
 TupleTools = "1"
 julia = "1.6"
+Combinatorics = "1"
```

Comment:
Please move this to the top of the `[compat]` section so the package names are alphabetized (by convention, the compat entry for `julia` stays at the bottom).